src/main/resources/templates/dashboard.html

```diff
@@ -203,4 +199,49 @@
 		</div>
 	</div>
 </div>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.6.0/Chart.bundle.min.js"></script>
+<script>
+	(function() {
+		window.onload = function() {
+			let ctx = document.getElementById('dashboard-active-hours');
+			let myChart = new Chart(ctx, {
+				type: 'line',
+				data: {
+					labels: ["Wed", "Thu", "Fri", "Sat", "Sun", "Mon", "Tues"],
+					datasets: [{
+						label: 'Evaluating',
+						backgroundColor: '#533e8d',
+						borderColor: '#533e8d',
+						data: [ 1.2, 0, 2.4, 1.1, 0.5, 0, 1.3 ],
+						fill: false,
+					}, {
+						label: 'Activity',
```

Comment:
How is activity determined? (For reference in the demo) Wouldn't evaluation time also be included in the activity calculation?